### PR TITLE
[ADD] 'mrp_operations' analysis;

### PR DESCRIPTION
**Pull request by legalsylvain**
My PoV : Nothing to do for 'mrp_operations';

### CODE REVIEW

File: addons/mrp_operations/migrations/8.0.1.0/openupgrade_analysis_work.txt [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---Fields in module 'mrp_operations'---
+mrp_operations / mrp.production.workcenter.line / production_state (selection)  : selection_keys is now '['cancel', 'confirmed', 'done', 'draft', 'in_production', 'ready']' ('['cancel', 'confirmed', 'done', 'draft', 'in_production', 'picking_except', 'ready']')
+# 'picking_except' key has been deleted. This key was unused in previous V7 version : Nothing to do
```
Comment (pedrobaeza) [inline addons/mrp_operations/migrations/8.0.1.0/openupgrade_analysis_work.txt:3]: Checked that this is dead code.

**Comment by pedrobaeza** [comment]
:+1:

**Comment by legalsylvain** [comment]
Again @pedrobaeza, thanks for your review !

**Comment by StefanRijnhart** [comment]
Thanks! :+1:

**Pull request merged**